### ISSUE TITLE
Alternative to sed regexp

### ISSUE THREAD

**Issue by dxcSithLord** [wagoodman/dive#514]
Not a bug, but have you considered, for readability,
```shell
curl -sL "https://api.github.com/repos/wagoodman/dive/releases/latest" | \
     grep '"tag_name":' | \
     awk -F'"' '{print $(NF-1)}'
```
with awk instead of using sed?  The awk statement uses quote (") as the field separator, making the version one less than the NF (number of fields)  Also use "\" to wrap the lines so they can be seen more easily.  The "\" must be followed by return and be the last character on the line (in case you did not know).

https://github.com/wagoodman/dive/blob/925cdd86482edec42185794620a1e616b79bbee5/README.md?plain=1#L100-L115

Just a suggestion.

**Comment by wagoodman** [comment]
I agree with the overall ask: it should be easier to install dive. I think adding an install.sh script would be a nice add (I've been curating one over in anchore/syft for a while now that would be good to copy and paste). But I don't want to loose the more specific ask from this issue either, which is being able to select and download a specific deb file for installation. It could be that the install script is modified to use apt in certain cases or add the capability to download deb assets easily.
